src/boss/config.go

```diff
@@ -12,12 +12,13 @@ import (
 var Conf *Config
 
 type Config struct {
-	Platform   string             `json:"platform"`
-	Scaling    string             `json:"scaling"`
-	API_key    string             `json:"api_key"`
-	Boss_port  string             `json:"boss_port"`
-	Worker_Cap int                `json:"worker_cap"`
-	Gcp        *cloudvm.GcpConfig `json:"gcp"`
+	Platform   string                   `json:"platform"`
+	Scaling    string                   `json:"scaling"`
+	API_key    string                   `json:"api_key"`
+	Boss_port  string                   `json:"boss_port"`
+	Worker_Cap int                      `json:"worker_cap"`
+	Gcp        *cloudvm.GcpConfig       `json:"gcp"`
+	Local      *cloudvm.LocalPlatConfig `json:"local"`
```

Comment:
Is there a motivation for a having a ptr to the sub configs instead of just values?